starters/next-drupal-starter/README.md

```diff
@@ -1,18 +1,19 @@
-# Next Drupal Starter
-
-## Demo Next.js Project setup
+# Pantheon Decoupled Kit Next Drupal Starter
 
 1. Clone this repo:
-```
-git clone git@github.com:pantheon-systems/decoupled-drupal-frontend-demo.git
+
+```bash
+git clone git@github.com:pantheon-systems/next-drupal-starter.git
```

Comment:
Is there any reason we'd want to use create-next-app here instead of a clone? Or maybe we list both options?

Reply:
Yeah good call. I'll add a create-next-app option.